.github/workflows/reusable-jira-workflow.yml

```diff
@@ -51,10 +48,19 @@ jobs:
         Write-Host "Branch name: $BranchName"
         "branch_name=$BranchName" >> $env:GITHUB_OUTPUT
 
+    - name: Extract Jira Issue
+      id: extract_jira
+      uses: workleap/wl-reusable-workflows/extract-jira-issue@feature/feng-1082_add_test_for_jira_validation
```

Comment:
Need to be put to @main.

(but would break this PR since does not already exist)